packages/affiliates/libs/synthPrices.js

```diff
@@ -9,7 +9,14 @@ module.exports = ({ web3, cryptowatchApiKey, tradermadeApiKey, decimals = 18 } =
     from = from / 1000;
     to = to / 1000;
     const priceFeed = await createReferencePriceFeedForFinancialContract(
-      winston.createLogger({ transports: [new winston.transports.Console()] }),
+      winston.createLogger({
+        transports: [
+          new winston.transports.Console({
+            level: "error",
+            stderrLevels: ["emerg", "alert", "crit", "error"]
```

Comment:
I dont think you need to define the log levels above the log you are curious in. you should be able to just define `error` and it will log everything above that.